<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>343</width>
    <height>169</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ZeroOMS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zerooms.qrc">
    <normaloff>:/resource/lightning.png</normaloff>:/resource/lightning.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>state</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>starter</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExecute"/>
   <addaction name="actionOrderlist"/>
   <addaction name="actionDigitView"/>
   <addaction name="actionPositionView"/>
  </widget>
  <action name="actionExecute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zerooms.qrc">
     <normaloff>:/resource/shield.png</normaloff>
     <normalon>:/resource/lightning.png</normalon>:/resource/shield.png</iconset>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="actionOrderlist">
   <property name="icon">
    <iconset resource="zerooms.qrc">
     <normaloff>:/resource/book_keeping.png</normaloff>:/resource/book_keeping.png</iconset>
   </property>
   <property name="text">
    <string>OrderList</string>
   </property>
  </action>
  <action name="actionDigitView">
   <property name="icon">
    <iconset resource="zerooms.qrc">
     <normaloff>:/resource/dollar.png</normaloff>:/resource/dollar.png</iconset>
   </property>
   <property name="text">
    <string>DigitView</string>
   </property>
  </action>
  <action name="actionPositionView">
   <property name="icon">
    <iconset resource="zerooms.qrc">
     <normaloff>:/resource/Settings-Location-icon.png</normaloff>:/resource/Settings-Location-icon.png</iconset>
   </property>
   <property name="text">
    <string>PositionView</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="zerooms.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExecute</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_ToggleExecute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_StartXingDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOrderlist</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_TriggerOrderList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_ToggleExecute()</slot>
  <slot>slot_StartXingDlg()</slot>
  <slot>slot_TriggerOrderList()</slot>
 </slots>
</ui>
